### PR TITLE
Rdm 10970

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
RDM-10970: Migration all CCD docker to postgres V11
        [RDM-10970] (https://tools.hmcts.net/jira/browse/RDM-10970).

### CODE REVIEW

File: PostgresV11.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+### V11 postgres migration steps
+
+##  Pull latest ccd docker
+* Synch all micro-services with the same branch version for instance: develop
```
Comment (mario-paniccia) [inline PostgresV11.md:4]:
```suggestion
* Make sure all micro-services running the same branch, for instance: 'develop'
```

File: PostgresV11.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+* Start ccd docker and make sure that the new container ccd-shared-database-v11 is up and running.
+* uncomment ccd-shared-database-v11 section in the backend.yml
+````
+#This section should be commented when the pr is merged in master.
+  ccd-shared-database-v11:
```
Comment (mario-paniccia) [inline PostgresV11.md:9]: ? 
don't we have to comment the old db?

Comment (Thor-tech-of-metal) [reply PostgresV11.md:9]: Yes , I could comment the old DB for this PR  and then we merge it  I comment the new DB, uncomment the old DB and comment the CCD_POSTGRES_11 VAR.  Should I do that ?

File: PostgresV11.md (hunk 1 above)
Comment (Thor-tech-of-metal) [reply PostgresV11.md:4]: DONE

File: PostgresV11.md (hunk 2 above)
Comment (Thor-tech-of-metal) [reply PostgresV11.md:9]: The comments "#This section should be commented when the pr is merged in master." is a reminder for me that we should not merge in develop ccd-shared-database-v11: section .  

I removed this comments from there .  I am going to add these notes in another place

**Comment by Thor-tech-of-metal** [comment]
REMEMBER THAT ,  ccd-shared-database-v11: in backend.yml should be commented when we merge this PR !

File: PostgresV11.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+* copy the dumpfile to the V11 container DB
+```
+docker cp dumpfile 36a8eb5cccba:/home
+```
+* Import dumpfile in to your new DB V11
+```
+bash the container
+docker exec -it 36a8eb5cccba bash
+
+change permission to your dumpfile file
```
Comment (mario-paniccia) [inline PostgresV11.md:49]: are you missing the `*` here and in some of the lines below?

Comment (Thor-tech-of-metal) [reply PostgresV11.md]: DONE

File: PostgresV11.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+Check db data
+psql
+SELECT datname FROM pg_database;
+\dt
+select * from event;
+````
+##  Settings
+* add CCD_POSTGRES_11 in your bash fie
+* export CCD_POSTGRES_11=ccd-shared-database-v11
```
[the remark this answers is not in the document]
Comment (Thor-tech-of-metal) [reply PostgresV11.md]: DONE

**Review by kiran-yenigala-hmcts — APPROVED** [review]
LGTM

**Review by rafalkalita — APPROVED** [review]

**Pull request merged**